stories/MewSwitch/MewSwitch.stories.js

```diff
@@ -21,7 +22,15 @@ export const MEWSwitch = () => ({
     },
     label: {
       default: text('label', 'Label')
+    },
+    value: {
+      default: boolean('is switch on?', false)
+    },
+    switchToggle: {
+      default: button('Toggle switch', () => {console.log('clicked'); return false})
```

Comment:
sorry for these can you just name them after the prop! (just so the user knows which prop they are playing with)